libraries/aws/common/src/aws_iot_subscription.c

```diff
@@ -77,7 +77,11 @@ static IotMqttError_t _modifySubscriptions( AwsIotMqttFunction_t mqttOperation,
     subscription.callback.pCallbackContext = NULL;
     subscription.callback.function = pSubscriptionInfo->callbackFunction;
 
-    /* Call the MQTT operation function. */
+    /* Call the MQTT operation function.
+     * Subscription count is 1 in this case.
+     * None of the flags are set in this call. Hence 0 is passed for flags.
+     * Please refer to documentation for AwsIotMqttFunction_t for more details.
```

Comment:
Note that [`AwsIotMqttFunction_t` is not currently documented](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/v4_beta/libraries/aws/common/include/aws_iot.h#L111), and that even if it was we don't have a reference to the doc entry here.  Those should be added, *and* we should also add comments for the 3rd and 4th arguments here explaining that these are the # of subscription and flags.

Reply:
It is already handled in this PR